src/main.ts

```diff
@@ -22,25 +22,23 @@ async function run(): Promise<void> {
     const githubClient = github.getOctokit(githubToken)
     const prNumber = github.context.issue.number
     const branchNameBase = github.context.payload.pull_request?.base.ref
+    const baseBranchSha = github.context.payload.pull_request?.base.sha
     const branchNameHead = github.context.payload.pull_request?.head.ref
     const useSameComment = JSON.parse(core.getInput('useSameComment'))
     const commentIdentifier = `<!-- codeCoverageDiffComment -->`
     const deltaCommentIdentifier = `<!-- codeCoverageDeltaComment -->`
     let commentId = null
+    execSync('/usr/bin/git fetch origin main')
+    execSync('/usr/bin/git checkout --progress main')
```

Comment:
Is this necessary to ensure we can detect which files have changed since main?

Reply:
Yep, this is not needed now, removed! since in FE workflow I added a step:

```
      - name: Fetch branches
        run: git fetch origin main
```